nsv13/code/modules/power/stormdrive.dm

```diff
@@ -532,102 +544,87 @@ Control Rods
 		input_power_modifier = chamber_ipm_total / reaction_rate
 
-		var/chamber_cooling_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/nitrous_oxide) * HIGH_COOLING + \
-													reaction_chamber_gases.get_moles(/datum/gas/water_vapor) * HIGH_COOLING + \
-													reaction_chamber_gases.get_moles(/datum/gas/hypernoblium) * HIGH_COOLING - \
+		var/chamber_cooling_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/hypernoblium) * VERY_HIGH_COOLING + \
+													reaction_chamber_gases.get_moles(/datum/gas/nitrogen) * HIGH_COOLING + \
+													reaction_chamber_gases.get_moles(/datum/gas/carbon_dioxide) * HIGH_COOLING - \
 													reaction_chamber_gases.get_moles(/datum/gas/tritium) * LOW_COOLING - \
 													reaction_chamber_gases.get_moles(/datum/gas/nucleium) * LOW_COOLING - \
 													reaction_chamber_gases.get_moles(/datum/gas/stimulum) * LOW_COOLING
 		cooling_power_modifier = chamber_cooling_total / reaction_rate
 
 		var/chamber_radiation_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/tritium) * HIGH_RADIATION + \
 													reaction_chamber_gases.get_moles(/datum/gas/nucleium) * HIGH_RADIATION - \
-													reaction_chamber_gases.get_moles(/datum/gas/pluoxium) * LOW_RADIATION
+													reaction_chamber_gases.get_moles(/datum/gas/bz) * LOW_RADIATION
 		radiation_modifier = chamber_radiation_total / reaction_rate
 
-		var/chamber_reinforcement_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/tritium) * HIGH_REINFORCEMENT + \
-														reaction_chamber_gases.get_moles(/datum/gas/pluoxium) * HIGH_REINFORCEMENT - \
-														reaction_chamber_gases.get_moles(/datum/gas/carbon_dioxide) * LOW_REINFORCEMENT - \
+		var/chamber_reinforcement_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/pluoxium) * VERY_HIGH_REINFORCEMENT + \
+														reaction_chamber_gases.get_moles(/datum/gas/tritium) * HIGH_REINFORCEMENT + \
+														reaction_chamber_gases.get_moles(/datum/gas/nitrous_oxide) * HIGH_REINFORCEMENT - \
 														reaction_chamber_gases.get_moles(/datum/gas/nucleium) * LOW_REINFORCEMENT - \
-														reaction_chamber_gases.get_moles(/datum/gas/stimulum) * LOW_REINFORCEMENT
+														reaction_chamber_gases.get_moles(/datum/gas/stimulum) * LOW_REINFORCEMENT - \
+														reaction_chamber_gases.get_moles(/datum/gas/bz) * LOW_REINFORCEMENT
 		reactor_temperature_modifier = chamber_reinforcement_total / reaction_rate
 
+		var/chamber_degradation_total = reaction_rate + reaction_chamber_gases.get_moles(/datum/gas/plasma) * HIGH_DEG_PROTECTION + \
+														reaction_chamber_gases.get_moles(/datum/gas/nitrous_oxide) * HIGH_DEG_PROTECTION + \
+														reaction_chamber_gases.get_moles(/datum/gas/hypernoblium) * HIGH_DEG_PROTECTION + \
+														reaction_chamber_gases.get_moles(/datum/gas/pluoxium) * HIGH_DEG_PROTECTION
```

Comment:
Doesn't this mean that adding these gases increases the rate the control rods degrade?